LayoutTests/fast/scrolling/mac/keyboard-scrolling-terminates.html

```diff
@@ -15,7 +15,7 @@
         async function runTest()
         {
             if (window.eventSender) {
-                eventSender.monitorWheelEvents();
+                await UIHelper.startMonitoringWheelEvents();
                 eventSender.keyDown("downArrow");
 
                 setTimeout(function() {
```

Comment:
Can we use UIHelper.waitForScrollCompletion here as well?

Reply:
Maybe; I was reluctant to make changes that might affect test behavior.